swcc/swcc/models/project.py

```diff
@@ -308,6 +309,7 @@ def load_analysis_from_json(self, file_path):
                         modes=modes,
                         charts=contents['charts'],
                         groups=[],
+                        good_bad_angles=contents['good_bad_angles'],
                     ).create()
```

Comment:
I know this suggestion isn't related to this PR, but since you had to make this change in two places it made me realize we can be more DRY here. Instead of writing the same `CachedAnalysis` constructor twice, can we just do one? Could you change it to be more like this?
```
groups_cache = []
if contents['groups']:
   ...
return CachedAnalysis(
   ...
   groups=groups_cache,
   good_bad_angles=contents['good_bad_angles'],
).create()
```

Reply:
Thank you for fixing this!